docs/ops.html

```diff
@@ -1826,111 +1857,25 @@ <h5><a id="kafka_streams_node_monitoring" href="#kafka_streams_node_monitoring">
         <th>Description</th>
         <th>Mbean name</th>
       </tr>
-      <tr>
-        <td>process-latency-avg</td>
-        <td>The average process execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>process-latency-max</td>
-        <td>The maximum process execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>punctuate-latency-avg</td>
-        <td>The average punctuate execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>punctuate-latency-max</td>
-        <td>The maximum punctuate execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>create-latency-avg</td>
-        <td>The average create execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>create-latency-max</td>
-        <td>The maximum create execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>destroy-latency-avg</td>
-        <td>The average destroy execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>destroy-latency-max</td>
-        <td>The maximum destroy execution time in ns. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
-      </tr>
       <tr>
         <td>process-rate</td>
-        <td>The average number of process operations per second. </td>
-        <td>kafka.streams:type=stream-processor-node-metrics,client-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
+        <td>The average number of records processed by a source processor node per second.</td>
```

Comment:
It's better to add a note at the beginning of `Processor Node Metrics` that the following four metrics are only available in certain types of nodes, e.g. process-rate and process-total would only be available for source nodes.

Reply:
Done